### PR TITLE
modify the meeting schedule for Europe and America area

### PR DESCRIPTION
**Pull request by Thor-wl**
Signed-off-by: Thor-wl <13164644535@163.com>

### CODE REVIEW

**Review by hwdef — COMMENTED** [review]
/lgtm

File: README.md [hunk 1]
```diff
@@ -141,9 +141,8 @@ kubectl create -f _output/release/volcano-monitoring-latest.yaml
 
 Community weekly meeting for Asia: 15:00 - 16:00 (UTC+8) Friday. ([Convert to your timezone.](https://www.thetimezoneconverter.com/?t=10%3A00&tz=GMT%2B8&))
 
-Community biweekly meeting for America: 08:30 - 09:30 (UTC-8) Thursday. ([Convert to your timezone.](https://www.thetimezoneconverter.com/?t=10%3A00&tz=GMT%2B8&))
-
-Community biweekly meeting for Europe: 11:00 - 12:00 (UTC+1) Thursday. ([Convert to your timezone.](https://www.thetimezoneconverter.com/?t=10%3A00&tz=GMT%2B8&))
+Community meeting for America and Europe is ongoing on demand now. If you have some ideas or topics to discuss, please leave
+message in the [slack](https://volcano-sh.slack.com). Maintainers will contact with you and book an open meeting for that.
```
Comment (hwdef) [inline README.md:145]: I can't access volcano's slack channel through this link, I don't know if it's my fault, or the link is dead.

**Review by hwdef — COMMENTED** [review]
/lgtm

File: README.md [hunk 2]
```diff
@@ -141,9 +141,8 @@ kubectl create -f _output/release/volcano-monitoring-latest.yaml
 
 Community weekly meeting for Asia: 15:00 - 16:00 (UTC+8) Friday. ([Convert to your timezone.](https://www.thetimezoneconverter.com/?t=10%3A00&tz=GMT%2B8&))
 
-Community biweekly meeting for America: 08:30 - 09:30 (UTC-8) Thursday. ([Convert to your timezone.](https://www.thetimezoneconverter.com/?t=10%3A00&tz=GMT%2B8&))
-
-Community biweekly meeting for Europe: 11:00 - 12:00 (UTC+1) Thursday. ([Convert to your timezone.](https://www.thetimezoneconverter.com/?t=10%3A00&tz=GMT%2B8&))
+Community meeting for America and Europe is ongoing on demand now. If you have some ideas or topics to discuss, please leave
```
Comment (william-wang) [inline README.md:144]: Let's keep America biweekly meeting. I'll work on that. thanks.

File: README.md [hunk 3]
```diff
@@ -141,9 +141,8 @@ kubectl create -f _output/release/volcano-monitoring-latest.yaml
...
+message in the [slack](https://cloud-native.slack.com/archives/C011GJDQS0N). Maintainers will contact with you and book an open meeting for that.
```
Comment (william-wang) [inline README.md:145]: agree with you :)

Comment (Thor-wl) [reply README.md:145]: OK.

File: README.md (hunk 2 above)
Comment (Thor-wl) [reply README.md:144]: OK. Thanks, William.

**Review by william-wang — APPROVED** [review]
/approve

**Pull request merged**